t/05-scheduler-iso.t

```diff
@@ -64,4 +64,14 @@ is($schema->resultset("GruTasks")->search({taskname => 'download_asset'}), 0, 'g
 map { like($_, $expected, 'expected warning') } @warnings;
 is($schema->resultset("GruTasks")->search({taskname => 'download_asset'}), 1, 'gru task should be created');
 
+# Using non-asset _URL does not create gru job and schedule jobs
+output_like { $ids = OpenQA::Scheduler::Scheduler::job_schedule_iso(DISTRI => 'opensuse', VERSION => '13.1', FLAVOR => 'DVD', ARCH => 'i586', NO_ASSET_URL => 'nonexistent.iso') } qr/_URL downloading only allowed for asset types!/, $warning, 'expected debug info about non-asset type';
```

Comment:
Right, missed that. I want to check stdout, so I readded Test::Output.
I thought that `warnings` call will catch all stderr outputs, but it cares only on `warn` and similar. That leads me to question if I shouldn't have used `warn` in 2a157ff instead of writing to stderr.